backload.rb

```diff
@@ -11,7 +11,9 @@ def load!
     next unless status == 'Closed' && group == 'Support'
     ticket_metrics = ticket.metrics
     requester = ticket.requester
-    next unless requester.external_id
```

Comment:
Let's be less aggressive in the `backload` function, since some users will not have external IDs for a variety of reasons, and we won't want to fail the entire script if a single user doesn't have an external ID. Something like the following would work:

```
unless requester.external_id
  puts "Failed to process ticket #{ticket.id}: no external ID"
  next
end
```